docs/source/tutorials/chat.rst

```diff
@@ -252,7 +253,7 @@ the Message dataclass.
 
 .. code-block:: python
 
-    def message_converter(sample: Mapping[str, Any]) -> List[Message]:
+    def message_converter(sample: Mapping[str, Any], train_on_input: bool) -> List[Message]:
```

Comment:
for this example I don't think it's necessary to add this parameter. but if you feel it's informative to keep it you should incorporate it in the logic below:
```
user_message = Message(
    role="user",
    content=input_msg,
    masked=not train_on_input,  # Mask if not training on prompt
)
```

Reply:
Done :heavy_check_mark: